setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name = "p4runtime",
+    version = "1.2.0",
+    packages = find_packages("py"),
+    install_requires = [
+        "protobuf>=3.5.0",
```

Comment:
how are the versions chosen? I am asking because they don't necessarily match the versions used by p4lang, and the version of protobuf here doesn't exactly match the version of protoc used to generate the Python files.

The version of gRPC required is also much more recent than the version of protobuf that's required. Maybe we could use `"grpcio >= 1.17"` so that the version used by the p4lang toolchain falls into the acceptable range?

Reply:
The version I am using is based on the version I got from the Stratum project. But I think it is fine to use the one you suggest. (grpcio 1.17.2 and protobuf 3.6.1 according to p4lang/third-party)